libpod/options.go

```diff
@@ -1754,6 +1754,18 @@ func WithPidFile(pidFile string) CtrCreateOption {
 	}
 }
 
+// WithInitCtrType indicates the container is a initcontainer
+func WithInitCtrType(containerType string) CtrCreateOption {
```

Comment:
In ctr_validate.go - can you validate that InitContainerType is only set if Pod is also set?